### PR TITLE
Feature/search queries

### PR DESCRIPTION
**Pull request by gallayl**

### CODE REVIEW

File: examples/sn-dms-demo/package.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
     "@types/lodash": "^4.14.119",
     "@types/lodash.groupby": "^4.6.4",
     "@types/node": "^10.12.12",
+    "@types/path-to-regexp": "^1.7.0",
```
Comment (B3zo0) [inline examples/sn-dms-demo/package.json]: path-to-regexp is not even a dependency of this package. I think that should be added as a dependency without the types package.

File: examples/sn-dms-demo/src/assets/picker.ts [hunk 2]
```diff
@@ -131,18 +131,18 @@ export const pickerTheme = createMuiTheme({
...
-        boxShadow: 'none',
-        '&.disabled-mobile-button': {
-          [theme.breakpoints.down('md')]: {
-            background: '#fff',
-            color: '#016D9E',
-            opacity: 0.25,
-          },
-        },
-      },
+      // '&$disabled': {
```
Comment (B3zo0) [inline examples/sn-dms-demo/src/assets/picker.ts]: Is this commented out code left there intentionally? If not I think there is no need to add commented out code to source control.

File: examples/sn-dms-demo/src/components/Search/SearchDocuments.tsx [hunk 3]
```diff
@@ -341,54 +336,64 @@ class SearchDocuments extends React.Component<
...
-                        </Grid>
-                        <Grid item={true} xs={3} />
-                        <Grid item={true} xs={titleWidth}>
-                          <Typography style={titleStyles} variant="body1">
-                            Contains
-                          </Typography>
-                        </Grid>
-                        <Grid item={true} xs={contentWidth}>
+                        </SearchRow>
+                        <SearchRow title="Contains">
```
[the remark this answers is not in the document]
Comment (herflis) [reply examples/sn-dms-demo/src/components/Search/SearchDocuments.tsx]: Move it to the resources.ts pls 👾

File: examples/sn-dms-demo/src/components/Search/SearchDocuments.tsx [hunk 4]
```diff
@@ -56,6 +56,45 @@ interface SearchDocumentsState {
   query?: string
 }
 
+const titleWidth = 2
+const contentWidth = 7
+const containerStyles: React.CSSProperties = {
```
[the remark this answers is not in the document]
Comment (gallayl) [reply examples/sn-dms-demo/src/components/Search/SearchDocuments.tsx]: We should stick to React.CSSProperties for DOM styling but in contentlist we are using inside a MuiTheme, therefore we have to use the one from @material-ui's typing. The two are not identical nor compatible (I think the latest "patch" release from material broke the compatibility 🤔)

[the remark this answers is not in the document]
Comment (gallayl) [reply examples/sn-dms-demo/src/components/Search/SearchDocuments.tsx]: reverted to React.CSSProperties

**Pull request merged**